app/routes.py

```diff
@@ -0,0 +1,279 @@
+        return {"message": f"Video {video_id} was not found"}, 404
+
+    elif request.method == "GET":
+        return {
+            "id": video.id,
+            "title": video.title,
+            "release_date": video.release_date,
+            "total_inventory": video.total_inventory
+        }, 200
+    elif request.method == "DELETE":
+        db.session.delete(video)
+        db.session.commit()
+
+        return jsonify({"id": video.id}), 200
+    elif request.method == "PUT":
+        form_data = request.get_json()
+        if "title" not in form_data:
+            return {"details" : "Invalid data"}, 400 
+        else:
+            video.title = form_data["title"]
+            video.total_inventory = form_data["total_inventory"]
+            video.release_date = form_data["release_date"]
+
+            db.session.commit()
+            return {"title" : video.title, "total_inventory" : video.total_inventory}, 200 
+
+@customers_bp.route("", methods=["GET", "POST"])
+def handle_customers():
+    if request.method == "GET":
+        customers_database = Customer.query.all() # searching for all the customers in the database 
+        customers_list = [] # empty list 
+        for each_customer in customers_database:
+                customers_list.append(
+                    {
+                    "name": each_customer.name,
+                    "id" : each_customer.id,
+                    "phone" : each_customer.phone,
+                    "registered_at" : each_customer.registered_at,
+                    "postal_code": each_customer.postal_code
+                })
+        return jsonify(customers_list), 200 
+
+    elif request.method == "POST": 
+        request_body = request.get_json()
+        if "postal_code" not in request_body:
+            return {"details": "Request body must include postal_code."}, 400
+        if "name" not in request_body:
+            return {"details": "Request body must include name."}, 400
+        if "phone" not in request_body:
+            return {"details": "Request body must include phone."}, 400
+        else:
+            new_customer = Customer(
+                name= request_body["name"],
+                phone=request_body["phone"],
+                postal_code =request_body["postal_code"])
+
+            db.session.add(new_customer)
+            db.session.commit()
+            new_customer_response = {
+                    "id": new_customer.id,
+                    "name" : new_customer.name,
+                    "phone": new_customer.phone,
+                    "registered_at": new_customer.registered_at,
+                    "postal_code": new_customer.postal_code
+                    }
+            return jsonify (new_customer_response), 201
+
+@customers_bp.route("/<customer_id>", methods=["GET", "DELETE", "PUT"])
+def handle_customer(customer_id):
+    if customer_id.isnumeric() != True:
+        return("Invalid Request"), 400
+    customer = Customer.query.get(customer_id)
```

Comment:
another way to handle this with a try/except. Like this:
```suggestion
   try:
        customer_id = int(customer_id)
    except ValueError:
        return f"Invalid customer id '{customer_id}'",400
    customer = Customer.query.get(customer_id)
```